ui/pages/confirm-add-suggested-token/confirm-add-suggested-token.component.js

```diff
@@ -136,24 +143,25 @@ export default class ConfirmAddSuggestedToken extends Component {
               type="secondary"
               large
               className="page-container__footer-button"
-              disabled={pendingTokens.length === 0}
-              onClick={() => {
-                addToken(pendingToken)
-                  .then(() => removeSuggestedTokens())
-                  .then(() => {
+              disabled={suggestedAssets.length === 0}
+              onClick={async () => {
+                await Promise.All(
```

Comment:
This will fail -- should be `Promise.all`

Reply:
whoa... thanks

Reply:
great catch hawk eye

Reply:
✅